src/org/zaproxy/zap/extension/ascanrules/ZapAddOn.xml

```diff
@@ -1,35 +1,37 @@
-<zapaddon>
```

Comment:
It seems that line endings were changed from ```\n``` to ```\r\n```.
Could you replace the line endings with the former?